src/main/java/com/troubleshooting/troubleshootingtool/ContainerDataRetrieval.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Retrieves the data returned from the Docker Client and packs the data in the appropriate data models
+ */
+package com.troubleshooting.troubleshootingtool;
```

Comment:
package name should come first

Reply:
and java docs before import statement?

Reply:
Java docs after import and before the class name.